src/gtimelog/tests/__init__.py

```diff
@@ -1,16 +1,23 @@
 """Tests for gtimelog"""
 import unittest
 
-from gtimelog.tests import test_timelog, test_settings, test_main
+from gtimelog.tests.core import test_utils as test_utils_core, test_reports, test_timelog, \
+    test_settings
+from gtimelog.tests.components import test_utils as test_utils_component
+from gtimelog.tests.modules import test_email
 
 
 def test_suite():
     return unittest.TestSuite([
-        test_timelog.test_suite(),
+        test_utils_core.test_suite(),
+        test_utils_component.test_suite(),
         test_settings.test_suite(),
-        test_main.test_suite(),
+        test_timelog.test_suite(),
+        test_reports.test_suite(),
+        test_email.test_suite(),
```

Comment:
I think it's time to switch to pytest.  This manual enumeration of test modules was fine when there were three test modules.

Reply:
I just thought the same thing just right now ^^

Reply:
I see that I already have `[tool:pytest]` configuration in setup.cfg, so perhaps all that's needed is changing tox.ini, Makefile, .travis.yml, appveyor.yml, and communicating with the Debian maintainer so he knows to add python3-pytest to Build-Depends and maybe adjust debian/rules accordingly.

Plus I'd want to make sure pytest discovers the exact same set of tests that python -m gtimelog.tests does.

In short, it's work.